upgrade-1.x.sh

```diff
@@ -350,8 +350,9 @@ function getSupervisorVersionFromRelease() {
     DEFAULT_SUPERVISOR_VERSION_URL="${DEFAULT_SUPERVISOR_VERSION_URL_BASE}images/${SLUG}/${HOSTOS_VERSION}/VERSION"
 
     # Get supervisor version for target resinOS release, it is in format of "a.b.c-shortsha", e.g. "4.1.2-f566dc4dd241",
+    # or newer "logsttream" supervisor, where the VERSIOn is "a.b.c-logstream", which is translated into "a.b.c_logstream",
```

Comment:
small typo `logsttream`

Reply:
another one: `VERSIOn`

Reply:
Fixed, @agherzan thanks, and sooooorrrrryyyy.... :)